### DISCUSSION TITLE
es2基板のU5-1部分の回路図誤記？

### DISCUSSION THREAD

**Discussion by y-ohe** [algyan/XIAOGYAN#24] [Q&A]
es2基板のスピーカー用オペアンプ部の回路図ですが、U5-1のピン4とピン2がショートしています。
誤記じゃないでしょうか？
基板のパターンとガーバーデータでは繋がっていないので、大丈夫だと思われます。
(DRCチェックでAirWireのエラーが出ます。)
![image](https://user-images.githubusercontent.com/18491913/217988971-27b427db-acf6-430b-bd03-86b83a7898db.png)

**Comment by matsujirushi** [comment]
ご連絡ありがとうございます！
Issue上げました。https://github.com/algyan/algyan_8th/issues/25

**Reply by matsujirushi** [reply]
訂正しました！#25

**Reply by y-ohe** [reply]
発注済みのPCBに影響がなく、回路図だけの誤記で良かったです！
